libyul/backends/evm/EVMObjectCompiler.cpp

```diff
@@ -90,8 +90,9 @@ void EVMObjectCompiler::run(Object& _object, bool _optimize)
 		);
 		if (!stackErrors.empty())
 		{
-			std::vector<FunctionCall*> memoryGuardCalls = FunctionCallFinder::run(
-				*_object.code,
+			auto const& code = *_object.code;
+			std::vector<FunctionCall const*> memoryGuardCalls = FunctionCallFinder::run(
+				code,
 				"memoryguard"_yulstring
```

Comment:
Any reason for the extra intermediate variable here? Does not seem to add much value.

Reply:
Oh, is that to force it to use the `const` version of `run()`? In that case I think that a cast would be clearer. Or naming it something like `constCode`.

Reply:
Always wanted to make use of `std::as_const`, this seems like a good place for it :D